docs/repositories-configure/integrations/github-integration.md

```diff
@@ -79,6 +79,32 @@ Adds AI-enhanced comments, providing insights and ready-to-commit AI-generated f
 
 ![AI-enhanced comment on GitHub](images/github-integration-ai-comment.png)
 
+### Automatic summaries
+
+Even when not written by developers, Codacy can provide an high-level summary for each pull request.
```

Comment:
I'm not a fan of this sentence but I don't have a good suggestion.

Reply:
```suggestion
Codacy can give a clear, high-level summary for each pull request, even if it's not written by developers.```

Reply:
Yeah that's better, I just have a tiny suggestion
```suggestion
Codacy can provide a clear, high-level summary for each pull request, even if one is not written by developers.
```